src/constants.ts

```diff
@@ -7,9 +7,7 @@ try {
 	_api = {
 		// @ts-ignore
 		runtime: {
-			...browser.runtime,
-			OnInstalledReason: chrome.runtime.OnInstalledReason,
-			restartAfterDelay: chrome.runtime.restartAfterDelay,
+			...browser.runtime
```

Comment:
> has this been tested on firefox and nothing has caused issues?

It shouldn't because we never use restartAfterDelay and chrome.runtime.OnInstalledReason should already be being  translated to browser.runtime.OnInstalledReason by Firefox according to [mdn](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities)

Reply:
(this was #271 for reference)